access/src/main/java/edu/unc/lib/dl/ui/controller/SearchActionController.java

```diff
@@ -73,13 +70,41 @@ public String search(Model model, HttpServletRequest request) {
 
     private String search(SearchRequest searchRequest, Model model, HttpServletRequest request) {
         SearchResultResponse resultResponse = doSearch(searchRequest, model, request);
+        String queryText = formatQueryText(request);
 
         model.addAttribute("resultType", "searchResults");
-        model.addAttribute("pageSubtitle", "Search Results");
+        model.addAttribute("pageSubtitle", "Search results for <span class=\"query-request\">\""
```

Comment:
I'd like to keep formatting/mark up out of our controllers. Maybe we could put the `queryText` into its model attribute and use it in `searchResults.jsp`